src/registry/date/index.stories.tsx

```diff
@@ -105,6 +105,50 @@ export const WithTooltipDatePicker: Story = {
   },
 };
 
+export const MultipleInputGroup: Story = {
```

Comment:
While looking in storybook at this story, I noticed the input group variant does not get focus whenever the "Add another" button is clicked (while the date picker variant does focus on the new input whenever the button is clicked). I'm not sure if we want to have consistent behavior for this (e.g clicking the button automatically focuses to the new input).

https://github.com/user-attachments/assets/42fda9a4-5886-4776-b1ba-693715b8cfd5

Reply:
Oh good one, I missed that! Autofocus should be achievable :-) will patch that